<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChromaControlDialog</class>
 <widget class="QDialog" name="ChromaControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Chromaticity</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>103</x>
     <y>248</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxOrbitClosure">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>13</y>
     <width>372</width>
     <height>226</height>
    </rect>
   </property>
   <property name="title">
    <string>Chromaticity </string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>46</y>
      <width>287</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>dp/p step size</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>no of samples (2n+1) </string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ScientificDoubleSpinBox" name="techSpinBoxStepSize">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinBoxNsamples">
       <property name="maximum">
        <number>250</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="radioButtonRing">
    <property name="geometry">
     <rect>
      <x>42</x>
      <y>190</y>
      <width>94</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ring Mode</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonLine">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>189</y>
      <width>94</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Line Mode</string>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScientificDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ScientificDoubleSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChromaControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChromaControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>497</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableRFFrequency()</slot>
 </slots>
</ui>
